docs/modules/ROOT/content-nav.adoc

```diff
@@ -5,4 +5,4 @@
 ** xref:getting-started/filters-and-projections.adoc[]
 ** xref:getting-started/relationships-and-advanced-filtering.adoc[]
 * xref:patterns.adoc[]
-* link:/cypher-builder/reference[Reference]
+// * link:/cypher-builder/reference[Reference]
```

Comment:
Is there any reason for disabling this link?

Locally it may not work, due to this being external docs, generated at publish time, but when published this points to https://neo4j.github.io/cypher-builder/reference/

Reply:
Yes, I was commenting out because it doesn't exist now, but then again since we're not making the docs public as of yet, it's not a problem. I will revert that!

Reply:
erm, well the docs are kinda unofficially published (https://neo4j.github.io/cypher-builder/cypher-builder/current/), and the page actually exists! (although outside of antora :sweat_smile: )

Reply:
Ah by published I meant indexed in the main documentation webpage! But I understand now the mistake that I made -- I was only building it locally, that's why it wasn't working :) Thanks for your review!